desdeo_tools/utilities/quality_indicator.py

```diff
@@ -26,9 +26,8 @@ def epsilon_indicator(reference_front: np.ndarray, front: np.ndarray) -> float:
     return eps
 
 
-@njit()
-def epsilon_indicator_ndims(reference_front: np.ndarray, front: np.ndarray) -> float:
-    """ Computes the additive epsilon-indicator between reference front and current approximating front.
+def epsilon_indicator_ndims(reference_front: np.ndarray, front: np.ndarray) -> list:
```

Comment:
Most other methods/classes in this package take the objective values (or front, in your case) as the first argument and the reference point (reference front in your case) as the second argument. This is not a big deal but reversing the order might be good.

Reply:
Moreover, you are just comparing one solution to another (I think) according to this line:
https://github.com/industrial-optimization-group/desdeo-emo/blob/442dcab702f2b97a9ac527837fc9c215788d8cef/desdeo_emo/EAs/IBEA.py#L95

So _do not_ call these fronts.

All of these indicators in general work with fronts. Epsilon indicator, in general, works with two fronts. Unless you test those cases, rename these variables something else.